gulpfile.babel.js

```diff
@@ -8,6 +8,9 @@ import watchify from 'watchify'
 import babelify from 'babelify'
 import envify from 'loose-envify/custom'
 import uglifyify from 'uglifyify'
+import path from 'path'
+import modulesify from 'css-modulesify'
+import cssnext from 'postcss-cssnext'
```

Comment:
nitpicking: I'd keep the whole name and camelCase it as `cssModulesify`, especially since `modulesify` gives no hint what it does. Not sure between `cssnext`/`postcssCssnext`.